solarforecastarbiter/reference_forecasts/utils.py

```diff
@@ -350,3 +358,32 @@ def check_persistence_compatibility(observation, forecast, index):
             raise ValueError('index=True not supported for forecasts'
                              ' with run_length >= 1day')
     _check_instant_compatibility(observation, forecast)
+
+
+def _limit_persistence_run_time(data_start, max_run_time, forecast):
+    """Get a the last run time that would result in the
+    forecast producing points as limited by PERS_PT_LIMIT.
```

Comment:
Here it's `PERS_PT_LIMIT`, but `PERSISTENCE_PT_LIMIT` in code and other documentation. I might spell out `POINT` too. 

Don't we prefix some Arbiter env vars with SFA? Not sure how consistent we might be about that so don't know if this is really an issue.

Reply:
Replace doc string with:

Get the last run time.

Last run time is either ``max_run_time`` or the time that limits the forecast length to value set by ``SFA_PERISTENCE_POINT_LIMIT`` environment variable or ``DEFAULT_PERSISTENCE_POINT_LIMIT``.